ui/app/AppLayouts/Wallet/views/RightTabView.qml

```diff
@@ -32,7 +32,8 @@ Item {
             Layout.fillWidth: true
             Layout.preferredHeight: parent.height - footer.height
             onCurrentIndexChanged: {
-                RootStore.backButtonName = ((currentIndex === 1) || (currentIndex === 2)) ? qsTr("Assets") : "";
+                RootStore.backButtonName = ((currentIndex === 1) || (currentIndex === 2)) ?
+                            qsTr("Assets") : (currentIndex === 3) ? qsTr("Activity") : ""
             }
```

Comment:
I assume this will become a `switch` at some point, for now is ok :)

Reply:
done!